File: acmicpc/1977.py
from math import *
a = ceil(sqrt(int(input())))
b = floor(sqrt(int(input())))
r = 0
for i in range(a, b+1):
    r += i*i
if r != 0:
    print("{}\n{}".format(r, a*a))
else:
    print(-1)

File: acmicpc/1977.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
	int m, n;

	cin >> m >> n;
	m = ceil(sqrt(m));
	n = floor(sqrt(n));

	int ans = 0;
	for (int i=m; i<=n; ++i)
		ans += i*i;

	if (ans)
		cout << ans << '\n' << m * m;
	else
		cout << "-1\n";

	return 0;
}
